kedro/framework/cli/starters.py

```diff
@@ -498,6 +502,9 @@ def _get_extra_context(  # noqa: PLR0913
             prompts_required, cookiecutter_context
         )
 
+    if starter_alias:
+        extra_context["starter_alias"] = starter_alias
```

Comment:
I'm not sure that it's a good idea to put starter_alias to the extra_context, because the extra context dict will be sent to cookiecutter where starter_alias is not needed?

Reply:
That's true, though this doesn't have any consequence I don't think it's the optimal approach here. Of the alternatives I do think it's the most minimal, but would opt for 1 in its stead. What do you think?

Reply:
Yes, I saw it was already done with some message for the same reason in lines 302-307), so it's not critical to do one more.
But maybe for consistency we should wrap all that messages that we throw after project creation in one func.